.travis.yml

```diff
@@ -30,6 +30,8 @@ script:
 - if [ "$OS" = "windows" ]; then bash "$TRAVIS_BUILD_DIR"/tools/travis/ci-build-windows-ms.sh; fi
 - if [ "$OS" = "linux" -a "$TRAVIS_PULL_REQUEST" != "false" ]; then bash "$TRAVIS_BUILD_DIR"/tools/travis/pr-build.sh; fi
 - cd "$TRAVIS_BUILD_DIR"
-- LUA_FILES=`find lua_modules lua_examples -iname "*.lua"`
-- echo checking $LUA_FILES
-- $LUACC -p $LUA_FILES
+- echo "checking:"
+- find lua_modules lua_examples -iname "*.lua" -print0 | xargs -0 echo
+- find lua_modules lua_examples -iname "*.lua" -print0 | xargs -0 $LUACC -p
+#- if [ "$OS" = "linux" ]; then bash "$TRAVIS_BUILD_DIR"/tools/travis/run-luacheck.sh; fi
```

Comment:
In my opinion it would make sense to change this line to
```- if [ "$OS" = "linux" ]; then bash "$TRAVIS_BUILD_DIR"/tools/travis/run-luacheck.sh || true ; fi```
for a start until all lua files are fixed. That way everybody can see the progress of fixing and what can be done.
It also makes sure that breaking the script can be detected early.

Reply:
This is a good idea, but I don't know if Travis will handle this many warnings:
`Total: 1177 warnings / 2 errors in 67 files`
I will try and revert this in case it fails.

Reply:
@HHHartmann 
>It does. I tried

Indeed! Build went fine and the warning list is visible in build log. Thanks again for reviewing this PR!